senza/traffic.py

```diff
@@ -395,3 +396,12 @@ def inform_sns(arns: list, message: str, region):
     sns = boto3.client('sns', region_name=region)
     for sns_topic in sns_topics:
         sns.publish(TopicArn=sns_topic, Subject="SenzaTrafficRedirect", Message=jsonizer.encode((message)))
+
+
+def resolve_to_ip_addresses(dns_name: str) -> set:
+    try:
+        answers = dns.resolver.query(dns_name, 'A')
+    except:
```

Comment:
If the 3rd party call gives a IO or OS error I would probably care about to know what is going on there since my code is somehow depending on the 3rd library. If the call is not important or if the library does not work, I would probably prefer to remove the dependency.

Even in one line the error can be from our side -- maybe we are passing a wrong value to the library and ignoring the error that would give me how to solve the issue. Silencing everything, in my opinion, does not sound good.